app/src/main/java/helper/ModulesAdapter.java

```diff
@@ -185,7 +189,21 @@ public void onClick(DialogInterface dialogInterface, int i) {
                                     mFileManager.downloadFile(module.getContents().get(0), module, courseName);
                                     break;
                                 case 1:
+                                    String toShare = "";
+                                    System.out.println(module.getContents());
```

Comment:
Remove `System.out.println`. Please use [timber](https://github.com/JakeWharton/timber) for logging and debugging.